integration-libs/digital-payments/src/checkout/cms-components/dp-payment-method/dp-payment-form/dp-payment-form.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="text-center">{{ 'dpPaymentForm.redirect' | cxTranslate }}</div>
+<div class="cx-spinner"><cx-spinner></cx-spinner></div>
+
+<ng-template #loading> </ng-template>
```

Comment:
Is it used?

Reply:
yes, this is used to show the message 'You will soon be redirected to third party page for card registration' before going to 3rd party page